payload.adoc

```diff
@@ -235,15 +235,14 @@ instruction context.
 [[sec:format33]]
 === Format 3 subformat 3 - Support
 
-This packet provides supporting information to aid the decoder. It is
-issued when
+This packet provides supporting information to aid the decoder.  It must be issued:
 
-* Trace is enabled or disabled;
-* The operating mode changes;
-* One or more trace packets cannot be sent (for example, due
-back-pressure from the packet transport infrastructure).
+* Before the first sync packet, in order to ensure the decoder is aware of how the encoder is configured.  It is recommended this be sent as soon as the encoder is enabled (*trTeEnable* changes from 0 to 1). This reduces the likelihood of having to generate two packets (support and sync-start) at the point tracing actually starts;
```

Comment:
This seems to change the specification. It was previously required to generate Support on trace enable but now its a recommendation. Its unclear why the clause for generating Support on operating mode is dropped.

Reply:
I don't believe it changes the spec, and it is certainly not my intention to do so.  Bear in mind that when the spec was ratified, the common control spec didn't exist, and there was no trTeEnable bit defined.  It was understood that there was some means of enabling the encoder but the precise mechanism was not defined.  There are now several aspects to enabling: trTeEnable is the overall enable, but trTeInstTracing must also be set in order to emit any decodable trace packets (e.g. a sync-start).   In the context of this section, the intended original meaning was "before any decodable trace is output" but this was not spelled out beyond "when enabled".  But now that the control is defined the original wording is not sufficiently precise.  What I'm attempting to do here is clarify that.
It's still mandatory to send a support packet before the first decodable trace packet (i.e. a sync-start).  I'm simply recommending that it be sent as early as possible after the encoder is enabled.

Reply:
@ved-rivos I have revised the wording - are you happy with this now?